run_tests.py

```diff
@@ -116,6 +115,10 @@ def test_seed(seed, interactive=True, repeat=1, retry=0):
                         help='Keep testing until some error is found.')
     parser.add_argument('--non-interactive', action='store_true',
                         help='Do not run gdb session if tests fail.')
+    parser.add_argument('--commands', nargs='*', metavar='CMD',
```

Comment:
After a while noone will remember another option to `run_tests.py`. It's much better to provide good defaults.